pkgs/applications/graphics/tesseract/wrapper.nix

```diff
@@ -21,6 +21,8 @@ let
     buildCommand = ''
       makeWrapper {$tesseractBase,$out}/bin/tesseract --set-default TESSDATA_PREFIX $out/share/tessdata
 
+      cp -n $tesseractBase/bin/* $out/bin/
```

Comment:
Why `-n`?

Reply:
To avoid overwrite the result of `makeWrapper {$tesseractBase,$out}/bin/tesseract`.

Reply:
Ah, that makes sense